extensions/amp-ad-network-dianomi-impl/0.1/dianomi-a4a-config.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @const @private {string} */
+const SRC_A4A_PREFIX_ = 'https://www.dianomi.com/smartads.pl?format=a4a';
+
+/**
+ * @param {!Window} win
+ * @param {!Element} element
+ * @return {boolean}
+ */
+export function dianomiIsA4AEnabled(win, element) {
```

Comment:
Where is this function being used?

Reply:
I am not calling this function in any of my implementations, this was added by following the network implementation guide (https://github.com/ampproject/amphtml/blob/main/ads/google/a4a/docs/Network-Impl-Guide.md#create-the-configuration-file).

If this is not required, should I remove the update from `ads/_a4a-config.js` as that was part of the 'Create configuration file' steps?

Reply:
I think the idea here is that you can determine the a4a extension based on this function (e.g. `'dianomi': dianomiIsA4AEnabled`) but since you always want to use it (`'dianomi': () => true`) I don't think this is necessary. 

@lannka to double-check me